conf/messages

```diff
@@ -1,4 +1,6 @@
 #COMMON
+service.name = Fulfilment House Due Diligence Scheme
```

Comment:
I think service name should be `Apply for the Fulfilment House Due Diligence Scheme`. what do you think?

Reply:
https://github.com/hmrc/play-frontend-hmrc#getting-started

Reply:
If you look at line no 4 in this file it is 

`app.name=Apply for the Fulfilment House Due Diligence Scheme`
Plus if you look into the service , the name in the header is Apply for the Fulfilment House Due Diligence Scheme